.github/workflows/muna-frontend.yml

```diff
@@ -0,0 +1,54 @@
+name: CI build
+
+on:
+  push:
+    branches:
+      - automation-test
+  pull_request:
+
+jobs:
+  build:
+
+    runs-on: ubuntu-18.04
+    strategy:
+      fail-fast: false
+      max-parallel: 6
+      matrix:
+        python-version: [3.7]
+        node-version: [12.x]
+
+    steps:
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v1
+      with:
+        python-version: ${{ matrix.python-version }}
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        pip install seleniumbase
+    - name: Install Chrome and Firefox
+      run: |
+        sudo apt install google-chrome-stable
+        sudo apt-get install firefox
+    - name: Check the console scripts interface
+      run: |
+        seleniumbase
+        sbase
+    - name: Install chromedriver and geckodriver (Firefox Webdriver)
+      run: |
+        seleniumbase install chromedriver
+        seleniumbase install geckodriver
+
+    - uses: actions/checkout@v2
+    - name: Use Node.js ${{ matrix.node-version }}
+      uses: actions/setup-node@v1
+      with:
+        node-version: ${{ matrix.node-version }}
+    - run: npm ci
+    - run: npm run build:test --if-present
+    - run: |
+        npm run serve-test &
+        npm run mock-server &
+        curl http://localhost:3004 -o /dev/null --connect-timeout 20 --retry 10 --retry-connrefused
+        curl http://localhost:3000 -o /dev/null --connect-timeout 20 --retry 10 --retry-connrefused
+        npm test
```

Comment:
can you use a node http server controlled from the test suite rather than the command line? then we can locally run tests and have the mock code update each time we run the test without having to look for the process id to kill it and restart it.

How do people normally run mock backend APIs for end-to-end tests?

Reply:
I don't quickly see a javascript API for http-server but there must be something like that

Reply:
I'm happy to merge before we fix this, but if it's quick to solve this now, it'd be great

Reply:
> can you use a node http server controlled from the test suite rather than the command line? then we can locally run tests and have the mock code update each time we run the test without having to look for the process id to kill it and restart it.
> 
> How do people normally run mock backend APIs for end-to-end tests?

I don't see a javascript api for http-server

Reply:
> How do people normally run mock backend APIs for end-to-end tests?

The options I know of are

1. MirageJS
2. Mockjax (For ajax requests)
3. Jest
4. Json-Server (What I currently use)

Reply:
Hi, @jbothma, I have successfully moved mock-server implementation to mocha test code, it works locally, pushed to github actions, build runs successfully.